### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by StoyanDimitrov**

### CODE REVIEW

File: guide/comments.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ logout = Logout { $user }
 }
 ```
 
-In FTL, Comments can be standalone or bound to a file, message, or section.
+In FTL, comments can be standalone or bound to a file, message, or section.
```
Comment (zbraniecki) [inline guide/comments.md]: In this case `Comment` is a name of the AST node, so it should stay upper-case.

File: guide/functions.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ targeted for such environments.
 
 ## Using Functions
 
-FTL Functions can only be called inside of placeables. Use them to return
+FTL functions can only be called inside of placeables. Use them to return
```
Comment (zbraniecki) [inline guide/functions.md]: Same here - `Function` is an AST node, should stay upper-case.

File: guide/functions.md [hunk 3]
```diff
@@ -152,14 +152,14 @@ log-time2 = Entry time: { DATETIME($date) }
 
 ## Partial arguments
 
-In most cases localizers don't need to call Functions explicitly, thanks to the
-implicit formatting. If the implicit formatting is not sufficient, the Function
+In most cases localizers don't need to call functions explicitly, thanks to the
+implicit formatting. If the implicit formatting is not sufficient, the function
```
Comment (zbraniecki) [inline guide/functions.md]: aanad here :)

**Comment by StoyanDimitrov** [comment]
Sorry, won't do. Don't have the time right now.

**Comment by stasm** [comment]
Sure, I understand. I took the liberty of fixing the issues @zbraniecki listed and I landed https://github.com/projectfluent/fluent/commit/dc8af1bced23f01fb838f3b15c338c59394de2b5, attributing you, @StoyanDimitrov. Thanks again!

**Pull request closed without merging**